rest/taskana-rest-spring-example/src/main/resources/application.properties

```diff
@@ -52,3 +52,5 @@ spring.resources.cache.cachecontrol.cache-private=true
 ####### tomcat is not detecting the x-forward headers from bluemix as a trustworthy proxy
 server.tomcat.internal-proxies=.*
 server.use-forward-headers=true
+####### allow bean overriding
+spring.main.allow-bean-definition-overriding=true
```

Comment:
What do we need this for?

Reply:
This is enabled in the `spring-boot-starter-parent`. Since I replaced the parent with our `taskana-rest-parent` I have to manually enable overriding. Having it disabled throws an error while starting the example app..

I just tried it out, somehow this is not necessary anymore..

Reply:
So you are going to remove it? Or do I get you wrong here?

If we still need it: what is the exception?

Reply:
yes. I will delete it.